data/risingwave/macros/get_power_data_and_groups.sql

```diff
@@ -0,0 +1,8 @@
+{% macro get_power_data_and_groups(table_name) -%}
```

Comment:
I'd prefer a name which is more explicit about the join happening. Perhaps `include_electrical_metadata` or `join_electrical_metadata`